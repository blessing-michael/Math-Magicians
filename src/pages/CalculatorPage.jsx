import React from 'react';
import { Calculator } from '../components';
import '../styles/App.css';

const CalculatorPage = () => (
  <div className="container">
    <div style={{ display: 'flex', justifyContent: 'space-between' }}>
      <h2>Lets do some Math</h2>
      <Calculator />
    </div>
  </div>
);

export default CalculatorPage;
