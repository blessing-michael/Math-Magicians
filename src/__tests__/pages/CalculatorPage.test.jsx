import React from 'react';
import { render } from '@testing-library/react';
import CalculatorPage from '../../pages/CalculatorPage';

describe('CalculatorPage', () => {
  test('Renders CalculatorPage and matches the snapshot', () => {
    const { container } = render(<CalculatorPage />);
    expect(container).toMatchSnapshot();
  });
});
