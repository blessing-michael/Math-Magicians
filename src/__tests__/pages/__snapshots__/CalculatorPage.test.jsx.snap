// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalculatorPage Renders CalculatorPage and matches the snapshot 1`] = `
<div>
  <div
    class="container"
  >
    <div
      style="display: flex; justify-content: space-between;"
    >
      <h2>
        Lets do some Math
      </h2>
      <div
        class="calc-container"
      >
        <h2
          class="intro-text"
        >
          Your Perfect Math Calculator
        </h2>
        <input
          disabled=""
          style="color: whitesmoke;"
          type="text"
          value="0"
        />
        <div
          class="all-btns"
        >
          <button
            class="grid-item undefined"
            type="button"
          >
            AC
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            +/-
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            %
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            รท
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            7
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            8
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            9
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            x
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            4
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            5
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            6
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            -
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            1
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            2
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            3
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            +
          </button>
          <button
            class="grid-item span-2"
            type="button"
          >
            0
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            .
          </button>
          <button
            class="grid-item undefined"
            type="button"
          >
            =
          </button>
        </div>
      </div>
    </div>
  </div>
</div>
`;
